src/app.rs

```diff
@@ -249,6 +255,10 @@ fn usage() -> HashMap<&'static str, Help> {
     doc!(h, "absolute-path"
         , "Show absolute instead of relative paths"
         , "Shows the full path starting from the root as opposed to relative paths.");
+    doc!(h, "path-separator"
+        , "Set the path separator to use when printing file paths."
+        , "Set the path separator to use when printing file paths. \
+           A path separator is limited to a single byte.");
```

Comment:
Maybe "limited to a single character" would be better for users that don't immediately associated 1 byte with 1 character. I know that it's probably not quite correct technically (multi-byte characters).

That makes me wonder: why not allow for general Unicode characters? I would change `u8` to `char` and change the help text to:

```
Set the path separator to use when printing file paths.
The separator must be a single character like '/' or '\'.
```